### DISCUSSION TITLE
Inquiry pay for plugin but can see it in the license to download

### DISCUSSION THREAD

**Discussion by lcmarval** [kimai/kimai#5155] [On-Premise: Installation & Updates]
I pay for the tasks plugin on friday, but i dont see it in the license page and dont see where else to ask or download it.
The transaction is in my bank account, not sure if the license needs more days to show in the web, all the other free plugings have been available at the moment of the request

**Comment by kevinpapst** [comment]
Hi, please send me an email to support@kimai.org

I see a purchase, but it has a different email attached and therefor the license management fails. 
Maybe a LemonSqueezy bug or did you change your email in the store?

Anyway: licenses are usually available immediately. But lets talk via email.
